layouts/page/pulumi-up.html

```diff
@@ -0,0 +1,34 @@
+{{ define "main" }}
+    <div class="pulumi-up-body text-white py-12">
+        <div class="container mx-auto">
+            <section id="about">
+                <div class="neon-container">
+                    <span class="blue-neon-sign-xl">Pulumi</span>
+                    <span class="green-neon-sign-xl">UP</span>
+                </div>
+                <pulumi-date-countdown
+                    value-label-class="text-2xl"
+                    text-class="w-full text-center text-white text-5xl date-countdown"
```

Comment:
I opted to just move those styles into `_pulumi_up.scss` because the Countdown component is really just text that changes and I don't really want to enforce any styles globally to it. It is more of a global component than anything (can be used anywhere) so I think it is good to enforce the style decisions to the user consuming it.